pytorch_lightning/trainer/trainer.py

```diff
@@ -2026,10 +2036,18 @@ def tpu_cores(self) -> int:
     def ipus(self) -> int:
         return self._accelerator_connector.num_ipus
 
+    @property
+    def hpus(self) -> int:
+        return self._accelerator_connector.num_hpus
+
```

Comment:
This will be replaced with parallel_devices_ids()

Reply:
Could you link the related PR or issue?

Reply:
Removed. Using `num_devices` instead.